sg_nomad/main.tf

```diff
@@ -0,0 +1,76 @@
+//
+// Module: tf_aws_sg/sg_nomad
+//
+// Nomad ports from https://www.nomadproject.io/docs/agent/config.html
+//
+
+// Provider specific configs
+provider "aws" {
+    access_key = "${var.aws_access_key}"
+    secret_key = "${var.aws_secret_key}"
+    region = "${var.aws_region}"
+}
+
+// Security Group Resource for Module
+resource "aws_security_group" "main_security_group" {
+    name = "${var.security_group_name}"
+    description = "Security Group ${var.security_group_name}"
+    vpc_id = "${var.vpc_id}"
+
+    // allows traffic from the SG itself for tcp
+    ingress {
+        from_port = 0
+        to_port = 65535
+        protocol = "tcp"
+        self = true
+    }
+
+    // allows traffic from the SG itself for udp
+    ingress {
+        from_port = 0
+        to_port = 65535
+        protocol = "udp"
+        self = true
+    }
+
+    // allow traffic for TCP 22 (SSH)
+    ingress {
```

Comment:
It was adapted from https://github.com/terraform-community-modules/tf_aws_sg/blob/master/sg_consul/main.tf#L29-L35 – I suppose that's a convenience if it is the only SG for that resource?